.github/workflows/test-check.yml

```diff
@@ -0,0 +1,78 @@
+name: Test Checker
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/*
+
+jobs:
+  test_check:
+    name: Check test results
+    runs-on: ubuntu-latest
+    steps:
+      - name: Wait for build to succeed ( ubuntu )
+        uses: tomchv/wait-my-workflow@v1.1.0
+        id: wait-for-build-ubu
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: "Tests & Build (ubuntu-latest)"
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          timeoutSeconds: 7200
+
+      - name: Make sure the build was a success ( ubuntu )
```

Comment:
nitpick: this step is marked as `skipped` (gray icon 🕧) when target workflow has succeded, so we can consider changing this step name:

```suggestion
      - name: Fail if the build was not a success ( ubuntu )
```

But it would be even more readable when we can check the condition and mark step as success (🟢 ).

Reply:
Good one! fixed it :)